content/en/references/persistence-mechanism.md

```diff
@@ -61,8 +61,6 @@ You can select a particular save strategy by setting `SNAPSHOT_SAVE_STRATEGY=<st
 * **`SCHEDULED`**: (**default**): saves at regular intervals the state of all the services that have been modified since the last snapshot.
```

Comment:
```suggestion
* **`SCHEDULED`** (**default**): saves at regular intervals the state of all the services that have been modified since the last snapshot.
```